src/main/java/net/imagini/zkcli/ZkDeleteHandler.java

```diff
@@ -0,0 +1,57 @@
+package net.imagini.zkcli;
+
+import org.apache.curator.framework.CuratorFramework;
+
+import java.util.Set;
+
+public class ZkDeleteHandler {
```

Comment:
Breaking this out into its own class is definitely a good idea for testability 👍